src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import './App.css';
+import PropTypes from 'prop-types';
 
-const App = () => (
-  <h1>React sum</h1>
+const App = ({ a, b }) => (
+  <h1>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
```suggestion
{
    `Sum of ${props.a} and ${props.b} is ${props.a + props.b}`
}
```

Reply:
Yeah, I tried to do this at first, but eslint doesn't think that it's OK to import React and then not use it, so it throws an error.